packages/core/docs/reference/migrate/2.5.11/overview.md

```diff
@@ -1,22 +1,24 @@
-# Migrating projects to 2.5.7
+# Migrating projects to 2.5.11
 
 ## Update `nuxt.config.js`
 
-In this release, we made the `middlewareUrl` property required for security reasons. Open the  `nuxt.config.js` file and add the `middlewareUrl` property like shown below:
+In this release, we've added the `middlewareUrl` and `ssrMiddlewareUrl` optional properties. You should use them to avoid security vulnerability.
+To enable this properties open the  `nuxt.config.js` file and add the `middlewareUrl`  and `ssrMiddlewareUrl` property in the `publicRuntimeConfig` object like shown below:
```

Comment:
```suggestion
In this release, we've added the `middlewareUrl` and `ssrMiddlewareUrl` optional properties. You should use them to avoid security vulnerability.
To enable these properties, open the  `nuxt.config.js` file and add the `middlewareUrl`  and `ssrMiddlewareUrl` properties to the `publicRuntimeConfig` object as shown below:
```